libs/core/src/lib/popover/popover-service/popover.service.ts

```diff
@@ -370,7 +378,7 @@ export class PopoverService extends BasePopoverClass {
     /** @hidden */
     private _applyWidthOverlay(): void {
         const maxWidthLimit = this.maxWidth ? this.maxWidth : MAX_BODY_SIZE;
-        const width = Math.min(this._getTriggerWidth(), maxWidthLimit);
+        const width = Math.min(this._getTriggerWidth(), maxWidthLimit) - WIDTH_OVERLAY_OFFSET;
```

Comment:
what is WIDTH_OVERLAY_OFFSET?

Reply:
I think we shouldn't use this hardcoded offset, instead we should rely on styles.
As far as I can see the original problem comes from the usage of `shadow` property and width on the same element.
We set min-width as popover-control width and set shadow style (mimic border) on the same node, which leads to such an effect.

In fundamnetal-styles we don't have this problem because examples there rely on `fd-popover_body` class and this class has a [modifier to align this thing](https://github.com/SAP/fundamental-styles/blob/9fed66352a4034aed39466714eb98af5c253c4da/src/popover.scss#L206-L210).
We need the same for `fd-popover__popper` but additionally we need to set `min-width/max-width` to the `fd-popover__popper` parent element instead of `fd-popover__popper` itself

Reply:
@dimamarksman you mean it's not a bug? or how we should deal with it?